Backend/ServerModules/ServerBehavior.cs

```diff
@@ -31,16 +31,36 @@ protected override void OnMessage(MessageEventArgs e)
             switch(Message.GetMessageType(rawString))
             {
                 case MessageType.TextMessage:
-                    var textMessage = new TextMessage(socket, rawString);
+                    if (user.IsMuted)
```

Comment:
hmm this is part of akrums ticket lakin ma 7nk just leave a comment in his ticket stating you already did this part

Reply:
Ah yikes I misunderstood that part my bad 🙏